Helper/LogOrganization.php

```diff
@@ -71,7 +72,9 @@ public static function organizeLogs() {
 
             $countCrit = 0;
         }
-        fclose($fp);
+        if ($fp !== null) {
```

Comment:
just remove fclose, this never be opened to be closed.

Reply:
Yes, on line 27:
https://github.com/facebookincubator/facebook-for-magento2/blob/main/Helper/LogOrganization.php#L27

Reply:
Yes, sorry. 
I this case, i think proper way to close it is, in the foreach, because you open 4 files and close only the last one.

Reply:
I just change it